go.mod

```diff
@@ -2,8 +2,9 @@ module github.com/deepgram-starters/text-to-speech-starter-go
 
 go 1.20
 
+require github.com/deepgram/deepgram-go-sdk v1.2.0
```

Comment:
Why this change?

Reply:
@naomi-lgbt  in all other cases we have the go SDK as a direct dependency not indirect, so I just aligned this project with that standard.